src/app/data-planes/components/data-plane-traffic/ServiceTrafficCard.vue

```diff
@@ -110,5 +125,11 @@ const props = defineProps<{
   display: flex;
   flex: 1 1 auto;
 }
-
+.title :deep(> a) {
+  text-decoration: none;
```

Comment:
**Question**: How come we’re removing the hover underline style here as well?

Reply:
Because this is what we do with all our clickable blocks. Actual in-text links should be blue with underline on hover. Clickable blocks shouldn't have titles that look like in-text links i.e. they should look like more like buttons not text-links.

Reply:
> Because this is what we do with all our clickable blocks.

Aren’t these the _only_ type of that kind of element?

> Clickable blocks shouldn't have titles that look like in-text links i.e. they should look like more like buttons not text-links.

But why? These titles don’t look like buttons (and rightfully so, they should absolutely not look like buttons) nor do they act like buttons. They act like links.

I guess what I’m saying is it’s not really clear why there is a special rule for links in certain UI fragments to have no underline on hover.

Reply:
> Aren’t these the only type of that kind of element?

- Table rows are clickable blocks
- Tabs are clickable blocks
- Dropdown MenuItems are clickable blocks
- I would guess we might have the odd button that is a a clickable block (KButton has an attribute for doing that at least https://alpha--kongponents.netlify.app/components/button.html#to). This is for making links look like buttons.

And now that you bring it up there are probably a few other blocks that we have that aren't currently linked but I think we should probably link them somewhere, plus there are quite a few blocks in Konnect that have clickable blocks and their titles don't underline on hover so I'm just following that styling as always.

Reply:
> I guess what I’m saying is it’s not really clear why there is a special rule for links in certain UI fragments to have no underline on hover.

Off the top of my head I think I would say:

"If the link is a textual link (i.e. within text) then it should be blue and underline on hover, otherwise its probably a UI 'button' therefore don't use the blue/underline on hover."

I guess there might be exceptions to that ^ but those will be clear exceptions not the general rule.

Reply:
I still think these rules are incredibly arbitrary and specifically it’s not clear why links that happen to be part of clickable boxes shouldn’t be underlined on hover. But of course I’m also of the opinion that links should generally be underlined (i.e. not just on hover) so this just follows from that.

Well, if you think that’s correct, then let’s go with that.

Reply:
Resolving this one.